services/frontend/src/components/ProfileCard/ProfileCard.js

```diff
@@ -0,0 +1,60 @@
+import {IconButton, Paper} from "@material-ui/core";
+import {ChevronRight, Email, LocalPhone} from "@material-ui/icons";
+import React from "react";
+import {AvatarIcon, SkillBadges} from "components/";
+import "./ProfileCard.scss";
+import {Link} from "react-router-dom";
+
+const DISPLAY_SKILL_COUNT = 3;
+
+const ProfileCard = ({id, name, primaryRole, contactEmail, phone, skills}) => {
+    // Only show the first DISPLAY_SKILL_COUNT skills, so as to not crowd the card too much
+    return (
+        <Paper className="people-profile-card">
+            <AvatarIcon
+                name={name}
+                personsRole=""
+                className="avatar-icon"
+            />
+
+            <div className="people-profile-card-details-section">
```

Comment:
This is basically the same component that is used in the `ProfileLayout`. Instead of it being copy-pasted, it should be refactored into a reusable component and shared between the two layouts.